src/app/comic/comic-list-container/comic-list-container.component.html

```diff
@@ -1,11 +1,27 @@
 <mu-header></mu-header>
-<div>
-  <button id = "add-comic">Add Comic</button>
+<div class="button-wrapper">
+  <button id="add-comic" (click)="onComicModalClick()">Add Comic</button>
 </div>
+<section class="modal-container" *ngIf="showComicModal">
+  <form (ngSubmit)="addComicDetail()" [formGroup]="modalComicFormGroup">
+    <div class="input-wrapper">
+      <input required class="modal-input-field" type="number" id="comicId" placeholder="Enter ComicID" formControlName="comicId">
+
+      <input required  class="modal-input-field" type="text" id="comicName" placeholder="Enter Comic Name"
```

Comment:
no form control for description?

Reply:
Actually I removed the description, It was not used, Since the card-list don't need description, and description is coming from API calls.